.github/util/package-install4j.sh

```diff
@@ -51,9 +52,12 @@ if [ ! -e target/keys ]; then
   echo "WARNING: not signing installer because target/keys directory does not exist."
   INSTALL4J_ARGS="$INSTALL4J_ARGS --disable-signing"
 else
-  INSTALL4J_ARGS="$INSTALL4J_ARGS --win-keystore-password=$INSTALL4J_WINDOWS_KEY_PASSWORD --mac-keystore-password=$INSTALL4J_APPLE_KEY_PASSWORD --apple-id=$INSTALL4J_APPLE_ID --apple-id-password=$INSTALL4J_APPLE_ID_PASSWORD"
+  INSTALL4J_ARGS="$INSTALL4J_ARGS --win-keystore-password=$INSTALL4J_WINDOWS_KEY_PASSWORD"
+  # Commenting out mac-related signing arguments
+  # INSTALL4J_ARGS="$INSTALL4J_ARGS --mac-keystore-password=$INSTALL4J_APPLE_KEY_PASSWORD --apple-id=$INSTALL4J_APPLE_ID --apple-id-password=$INSTALL4J_APPLE_ID_PASSWORD"
```

Comment:
Those flags are referenced in the install4j configuration: https://github.com/liquibase/liquibase/blob/8ef23d5c4a9a8fe56c8a71ff1ae1b0b97714923a/liquibase-dist/src/main/install4j/liquibase.install4j#L11
I believe you will need to update this file too to get rid of macos, but I'm not sure if just removing some tags or changing flags to false would get it done. Maybe you could: 
* download install4j https://www.ej-technologies.com/install4j
* load liquibase.install4j file
* disable macos installer using it and see what it changes in this XML

Reply:
I'll check on this.

Reply:
I loaded the liquibase.intall4j file and deleted the macOS from media. Changes are made in the PR as per the edits mades in the xml file after the macOS removal.